recipes/aicssegmentation/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "aicssegmentation" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicssegmentation-{{ version }}.tar.gz
+  sha256: 65a6e695ef0c5d4afe224e31259ae6cb8144aa474952a13c99d7426b2cc3f87b
+
+build:
+  entry_points:
+    - batch_processing=aicssegmentation.bin.batch_processing:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=5.2
+    - python >=3.7
+  run:
+    - aicsimageio >=4.0.5
+    - dask-core
+    - itk
+    - itk-meshtopolydata
+    - itkwidgets
+    - jupyter
+    - matplotlib-base
+    - numpy >=1.15.1
+    - pandas >=0.23.4
+    - python >=3.7
+    - scikit-image >=0.18.0
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - aicssegmentation
+  commands:
+    - pip list
+    # - pip check
+    - batch_processing --help
```

Comment:
I would consider using a different (more specific) name for the command line executable, but that's outside the scope of this review :)